server/protocols/jmap/src/main/java/org/apache/james/jmap/model/mailbox/Role.java

```diff
@@ -34,14 +36,14 @@
 
     public static final String USER_DEFINED_ROLE_PREFIX = "x-";
     
-    public static final Role INBOX = new Role("inbox");
-    public static final Role ARCHIVE = new Role("archive");
-    public static final Role DRAFTS = new Role("drafts");
-    public static final Role OUTBOX = new Role("outbox");
-    public static final Role SENT = new Role("sent");
-    public static final Role TRASH = new Role("trash");
-    public static final Role SPAM = new Role("spam");
-    public static final Role TEMPLATES = new Role("templates");
+    public static final Role INBOX = new Role(DefaultMailboxes.INBOX);
+    public static final Role DRAFTS = new Role(DefaultMailboxes.DRAFTS);
+    public static final Role OUTBOX = new Role(DefaultMailboxes.OUTBOX);
+    public static final Role SENT = new Role(DefaultMailboxes.SENT);
+    public static final Role TRASH = new Role(DefaultMailboxes.TRASH);
+    public static final Role ARCHIVE = new Role("Archive");
+    public static final Role SPAM = new Role("Spam");
+    public static final Role TEMPLATES = new Role("Templates");
```

Comment:
Then I rolled it back.
And I changed a bit on code with searching by expression. I assume that we will search ignore case sensitive such as mailbox's role was there so it does not make sense getting all mailbox then filter by name.
For further, if we need to support search with case sensitive than we can add more parameter on searching method.
WDYT?